var/spack/repos/builtin/packages/xtensor/package.py

```diff
@@ -34,15 +34,18 @@ class Xtensor(CMakePackage):
 
     version('develop', branch='master',
             git='https://github.com/QuantStack/xtensor.git')
+    version('0.15.1', 'c24ecc406003bd1ac22291f1f7cac29a')
     version('0.13.1', '80e7e33f05066d17552bf0f8b582dcc5')
 
     variant('xsimd', default=True,
             description='Enable SIMD intrinsics')
 
     depends_on('xtl')
-    depends_on('xtl@0.3.4:', when='@develop')
+    depends_on('xtl@0.4.0:', when='@develop')
+    depends_on('xtl@0.4.0:', when='@0.15.1')
```

Comment:
You can do ‘0.15.1:’ and develop will satisfy this

Reply:
that's true, I was keeping it in place in case it updates soon again so one has a clear point to modify when doing dev-testing. But can change!

Reply:
up to you, i just saw that in the current state you can turn 2 lines into 1